docs/en/modules/admin/pages/components/budgets.adoc

```diff
@@ -16,7 +16,7 @@ Some examples of where this component was used:
 
 * https://www.decidim.barcelona/processes/PressupostosParticipatius[2020/2021 Participatory Budgeting Process in Decidim Barcelona].
 The  city of Barcelona (Spain) has allocated a budget of 30.000.000 EUR so people can present, define, prioritize, vote
-and choose which investment projects the city goverment should execute.
+and choose which investment projects the city government should execute.
```

Comment:
Do you know why is that we're only seeing this and not all the other linter offences that I can see in the [jobs' output](https://github.com/decidim/documentation/actions/runs/6123228660/job/16620685356)? Is that a GH Action setting, or I'm missing something?

Reply:
I set it to not be so noisy, so it will only show you errors in the code that you actually change in the PR maybe +/- 1-2 lines iirc. We could make this different, but it would mean we would need to do a monster PR to fix all of the warnings, at least, to avoid annoying, the heck out of everyone!

Reply:
Ok! IMO this monster PR is typically when introducing new Linters, but we can do so after this PR